src/nanoemoji/nanoemoji.py

```diff
@@ -374,16 +391,22 @@ def write_bitmap_builds(
         nw.build(dest, "write_bitmap", rel_build(svg_file))
 
 
-def write_zopflipng_builds(
-    zopflipng_builds: Set[Path], nw: NinjaWriter, master: MasterConfig
+def write_compressed_bitmap_builds(
+    builds: Set[Path],
+    nw: NinjaWriter,
+    master: MasterConfig,
+    rule_name: str,
+    dest_dir: Path,
+    infile_fn: Callable[[Path], Path],
+    outfile_fn: Callable[[Path], Path],
 ):
-    os.makedirs(str(zopflipng_dir()), exist_ok=True)
+    os.makedirs(str(dest_dir), exist_ok=True)
```

Comment:
Optional, there's a whole set of required dirs created early in the build, could create this there

Reply:
but these are only required if any of the font configs has bitmaps and uses either zopflipng or pngquant, etc. better to create on demand i think